src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestIT.java

```diff
@@ -889,8 +889,32 @@ public void testNullValuesOnMultiDataTypes() throws Exception {
       verifyInsertValidationResponse(channel1.insertRow(row, Integer.toString(val)));
     }
 
+    // Sleep for a few seconds to make sure a flush
+    Thread.sleep(5000);
```

Comment:
Could you block on the flush finishing instead of sleeping?

Reply:
i'm not sure how to do that, we don't have a flush API on the public interface anymore